example/src/Examples/Graphs/Interpolation.tsx

```diff
@@ -25,28 +23,22 @@ export const Interpolation: React.FC<GraphProps> = ({ height, width }) => {
     createGraphPath(width, height, 60)
   );
 
-  const onPress = useCallback(() => {
-    setCurrentPath(createGraphPath(width, height, 60));
-  }, [height, width]);
+  // const onPress = useCallback(() => {
```

Comment:
Remove comments :)

Reply:
not sure I understand?

Reply:
I like the idea to show the non animated alternative